network-api/app/networkapi/landingpage/models.py

```diff
@@ -19,6 +19,12 @@ class Signup(models.Model):
         help_text="Body of signup component"
     )
 
+    newsletter = models.CharField(
+        max_length=100,
+        help_text="The (pre-existing) SalesForce newsletter to sign up for",
+        default="mozilla-leadership-network"
```

Comment:
do you want to allow them to leave the field blank and then in the db it defaults to this? Or is this just a default for existing entries and they always have to put something in for this field in the admin interface?

Reply:
The latter. Also the fact that new entries get pre-populated with this in the admin is helpful too.